src/server.c

```diff
@@ -1606,6 +1606,8 @@ void createSharedObjects(void) {
         "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
     shared.slowscripterr = createObject(OBJ_STRING,sdsnew(
         "-BUSY Redis is busy running a script. You can only call FUNCTION KILL or SHUTDOWN NOSAVE.\r\n"));
+    shared.slowmoduleerr = createObject(OBJ_STRING,sdsnew(
+        "-BUSY Redis is busy running a module command.\r\n"));
```

Comment:
This can be removed now I guess.

Reply:
i'll use it as a default.. i was actually paranoid about somehow reaching that code that uses it when in_busy_module is still true, and somehow already out of the scope of RM_Yield.